commit 4eb7f6ab362f4fff79f0c0e5b2147f2c5eb22687
Author: Aaron Seigo <aseigo@kde.org>
Date:   Sun Sep 29 18:08:53 2013 +0200

    konsole -> kwin
    
    long live copy and paste coding (i fixed this ages ago in the place this was
    borrowed from)

diff --git a/libs/taskmanager/launcheritem.cpp b/libs/taskmanager/launcheritem.cpp
index 8377a09..ac94732 100644
--- a/libs/taskmanager/launcheritem.cpp
+++ b/libs/taskmanager/launcheritem.cpp
@@ -299,7 +299,7 @@ QString LauncherItem::defaultApplication() const
         return browserApp;
     } else if (application.compare("terminal", Qt::CaseInsensitive) == 0) {
         KConfigGroup confGroup(KGlobal::config(), "General");
-        return confGroup.readPathEntry("TerminalApplication", QString::fromLatin1("konsole"));
+        return confGroup.readPathEntry("TerminalApplication", QString::fromLatin1("kwin"));
     } else if (application.compare("filemanager", Qt::CaseInsensitive) == 0) {
         KService::Ptr service = KMimeTypeTrader::self()->preferredService("inode/directory");
         if (service) {
